### PR TITLE
Manual Pages from Sphinx

### PR DESCRIPTION
**Pull request by jwrober**
Part of #1947 

Draft PR right now to get some feedback. To test locally, run this from the Freeciv21 source root directory: `sphinx-build -b man docs build/docs/man`. Then open the created man page with `man ./build/docs/man/freeciv21-server.6`

### CODE REVIEW

File: docs/Manuals/Program/freeciv21-server.rst [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+========
+
+Here are some examples:
+
+freeciv21-server --file oldgame.sav -p 2224
+  Starts a server on port `2224`, loading the savegame named `oldgame.sav`.
+
+freeciv21-server -R ranklog -l logfile -r start_script -f oldgame.sav.xz -p 2224
+  Starts a server on port `2224`, loading the savegame named `oldgame.sav.xz`. Player ranking events are
+  logged to `ranking`, other logging events are placed in `logfile`. When the server first starts, it
```
Comment (daavko) [inline docs/Manuals/Program/freeciv21-server.rst:62]: Should `ranking` say `ranklog` instead in this sentence? The command is using `ranklog`

File: docs/Manuals/Program/freeciv21-server.rst [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+  immediately executes the server settings script named `start_script`. See freeciv21-server-manual(6) for
+  more information on the server settings script.
+
+freeciv21-server -a -D users.conf -G -N -q 60  -l logfile -s ./saves -p 2224
```
Comment (daavko) [inline docs/Manuals/Program/freeciv21-server.rst:66]:
```suggestion
freeciv21-server -a -D users.conf -G -N -q 60 -l logfile -s ./saves -p 2224
```

**Review by daavko — COMMENTED** [review]
Just two small things, otherwise LGTM from me for the text side of things

File: docs/Manuals/Program/freeciv21-server.rst [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+  Starts a server on port `2224` with authentication enabled (`-a`, `-D`, `-G`, and `-N`). If no activity
+  for 60 seconds the server will quit (`-q`). Logging events are placed in `logfile` and periodic saves are
+  placed in the `./saves` directory.
+
+
+FILES
+=====
+
+The freeciv21-server requires the default ruleset files to be readable in the Freeciv21 `data` directory.
+This is most often `/usr/local/share/freeciv21`.
```
Comment (lmoureaux) [inline docs/Manuals/Program/freeciv21-server.rst:76]: Is this true for our debs? It's certainly not for snaps

File: docs/Manuals/Program/freeciv21-server.rst [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+* civ2civ3/buldings.ruleset
```
Comment (lmoureaux) [inline docs/Manuals/Program/freeciv21-server.rst:78]:
```suggestion
* civ2civ3/buildings.ruleset
```

File: docs/Manuals/Program/freeciv21-server.rst [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+* civ2civ3/techs.ruleset
+* civ2civ3/terrain.ruleset
+* civ2civ3/units.ruleset
+* civ2civ3/script.lua
+* default/default.lua
+* default/nationlist.ruleset
+
+These files are for the default ruleset for the game (civ2civ3), which are loaded if you do not use the
+`--ruleset` arguement. Alternate rules can be loaded with the ``rulsetdir`` directive in a start up script.
+Type ``help resetdir`` at the server command prompt for more information.
```
Comment (lmoureaux) [inline docs/Manuals/Program/freeciv21-server.rst:94]:
```suggestion
`--ruleset` arguement. Alternate rules can be loaded with the ``rulesetdir`` directive in a start up script.
Type ``help rulesetdir`` at the server command prompt for more information.
```

File: docs/Manuals/Program/freeciv21-server.rst [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+freeciv21-server -a -D users.conf -G -N -q 60 -l logfile -s ./saves -p 2224
+  Starts a server on port `2224` with authentication enabled (`-a`, `-D`, `-G`, and `-N`). If no activity
+  for 60 seconds the server will quit (`-q`). Logging events are placed in `logfile` and periodic saves are
+  placed in the `./saves` directory.
+
+
+FILES
+=====
+
+The freeciv21-server requires the default ruleset files to be readable in the Freeciv21 `data` directory.
```
Comment (lmoureaux) [inline docs/Manuals/Program/freeciv21-server.rst:75]: I found this a bit hard to understand. I would prefer something more generic.

File: docs/Manuals/Program/freeciv21-server.rst [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+  immediately executes the server settings script named `start-script`. See freeciv21-server-manual(6) for
+  more information on the server settings script.
+
+freeciv21-server -a -D users.conf -G -N -q 60 -l logfile -s ./saves -p 2224
+  Starts a server on port `2224` with authentication enabled (`-a`, `-D`, `-G`, and `-N`). If no activity
+  for 60 seconds the server will quit (`-q`). Logging events are placed in `logfile` and periodic saves are
+  placed in the `./saves` directory.
+
+
+FILES
```
Comment (lmoureaux) [inline docs/Manuals/Program/freeciv21-server.rst:72]: We should mention `database.lua` in this section.

File: docs/Manuals/Program/freeciv21-server.rst [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+  be changed from the default of UTF-8, since that is the format of the supplied rulesets and the standard
+  network protocol.
+
+FREECIV_INTERNAL_ENCODING
+  Sets the character encoding used internally by freeciv21-server. This encoding should not be visible at
+  any interface. Defaults to UTF-8.
+
+FREECIV_LOCAL_ENCODING
+  Sets the local character encoding (used for the command line and terminal output). The default is inferred
+  from other aspects of the environment.
```
Comment (lmoureaux) [inline docs/Manuals/Program/freeciv21-server.rst:121]: Don't advertise them. The data and internal encoding should always be UTF-8. The local encoding should be determined by Qt.

File: docs/Manuals/Program/freeciv21-server.rst [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+  (This does not affect where the server creates save game files. See the `--saves` option for that.)
+
+FREECIV_SCENARIO_PATH
+  A colon separated list of directories pointing to the Freeciv21 scenario directories. By default Freeciv21
+  looks in the following directories, in order, for scenario files: the current directory; the
+  "data/scenarios" subdirectory of the current directory; the subdirectory
+  ".local/share/freeciv21/scenarios" in the user's home directory; and the directory where the files are
+  placed by running "cmake --target install".
+
+  (This does not affect where the server creates scenario files See the `--scenarios` option for that.)
```
Comment (lmoureaux) [inline docs/Manuals/Program/freeciv21-server.rst:146]:
```suggestion
  (This does not affect where the server creates scenario files. See the `--scenarios` option for that.)
```

**Review by lmoureaux — COMMENTED** [review]
I like the approach. How do you envision integrating this with our build system?

**Comment by jwrober** [comment]
> I like the approach. How do you envision integrating this with our build system?

I think we will add python3 and sphinx to CI/CD. Build the files and then update install to include. Do it in maybe a couple PRs

**Pull request merged**